tests/fixtures/roles.references.numref.md

```diff
@@ -0,0 +1,43 @@
+Testing named figures and numbered references
```

Comment:
Can you add in the test an example of a ref/numref missing a target (e.g. `test5`)

Reply:
Done!